docs/conf/atlantic/2024/writing-day.rst

```diff
@@ -127,3 +127,17 @@ Project list
 ------------
 
 Submit your project using our `Writing Day project form <https://forms.gle/uTkWHV3fesyNQEyk9>`_ or send us a PR with your project info!
+
+How far is it to Jupyter? (Session 2)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Join project organizer, Mike O'Neill, during the second session of the day.
+
+Learn how to use `Jupyter <https://jupyter.org/>`__ to be a more effective technical writer. The goal of this Writing Day project is to help you understand how using Jupyter in your tool chain can assist you in creating docs.
```

Comment:
```suggestion
Learn how to use `Jupyter <https://jupyter.org/>`__, a web-based interactive development environment for notebooks, code, and data, to be a more effective technical writer. The goal of this Writing Day project is to help you understand how using Jupyter in your tool chain can assist you in creating docs.
```
Or something similar on what Jupyter actually is, would be good. I pulled this from the website, so it might be the wrong description.

Reply:
thanks @mxsasha, I'm hoping the project leader can clarify when I pull him into the review.

Reply:
I would agree with you because this is what you generally do: write it so that everyone can understand what you are talking about.
Now, 

1. Jupyter is nothing you stumble into.  You have to be expert in something to need using it.
2. among many things 'a web-based' 
3. 'interactive development environment' that is what makes it better than others

Reply:
Do you recommend that potential contributors have certain python and SQL packages downloaded before arriving at the event?

If so, @sciustechnologia, can you give me the versions and links to include?